src/main/java/sdu/se9/tv2/management/system/persistence/PersistencePerson.java

```diff
@@ -86,79 +79,19 @@ public Person getPerson (int personID) {
      * @param lastName Lastname to search for
      * @return
      */
-    public ArrayList<Person> getPersons (String firstName, String lastName) {
-        ArrayList<Person> persons = new ArrayList<Person>();
-
-        for (int i = 0; i < this.persons.size(); i++) {
-            Person element = this.persons.get(i);
-
-            if (element.getFirstName().equals(firstName) && element.getLastName().equals(lastName)) {
-                persons.add(element);
+    public ArrayList<Person> getPersons (String firstName, String lastName) throws SQLException {
+        Connection connection = PersistenceDatabaseHelper.getConnection();
+
+            PreparedStatement stmt = connection.prepareStatement("SELECT * FROM person WHERE first_name ='" + firstName + "' AND last_name ='" + lastName+"'");
```

Comment:
SQL injection. Dette kan rent faktisk bruges til at få fuld adgang til databasen, hvilket betyder man kan få og redigerer alt data, og f.eks. også slette hele databasen. Det kan man her fordi der er det strenge som brugeren selv skriver ind, og man kan derfor til f.eks. efternavnet skrive noget som "\"DROP TABLE...'" for at slette tabeller i databasen.